commands/import.go

```diff
@@ -121,10 +121,10 @@ func setRolling(dbExists bool, dbIsRolling bool, dbCurrChunk int, dbTotalChunks
 	}
 
 	if userTotalChunks != -1 { // user gave the total number of chunks via command line
-		// it's currently an error to try to change the total number of chunks in an existing rolling database
-		if dbExists && dbIsRolling && dbTotalChunks != userTotalChunks {
+		// it's currently an error to try to reduce the total number of chunks in an existing rolling database
+		if dbExists && dbIsRolling && userTotalChunks < dbTotalChunks  {
```

Comment:
Now the only thing that isn't allowed is decreasing the number of chunks in an already existing rolling database. This is because we have to decide how to handle the chunks that may already be in the database and need to be removed. Increasing isn't an issue as nothing is in those chunk positions yet.